src/__main__.py

```diff
@@ -1554,21 +1555,22 @@ def cleanup():
         print('This tool requires root permissions to run.')
         sys.exit(ExitCodes.CRITICAL)
     else:
-        for line in (
-            _(
-                'This command will collect system configuration and '
-                'diagnostic information from this system.'
-            ),
-            _(
-                'The generated archive may contain data considered sensitive '
-                'and its content should be reviewed by the originating '
-                'organization before being passed to any third party.'
-            ),
-            _(
-                'No changes will be made to system configuration.\n'
-            ),
-        ):
-            print('\n'.join(textwrap.wrap(line)))
+        if not '--quiet' in sys.argv:
+            for line in (
+                _(
+                    'This command will collect system configuration and '
+                    'diagnostic information from this system.'
+                ),
+                _(
+                    'The generated archive may contain data considered sensitive '
+                    'and its content should be reviewed by the originating '
+                    'organization before being passed to any third party.'
+                ),
+                _(
+                    'No changes will be made to system configuration.\n'
+                ),
+            ):
+                print('\n'.join(textwrap.wrap(line)))
```

Comment:
why not logging.info as you did above?

Reply:
I was not changing existing code in this patch, just adding into a if statement. We can replace with logging.info here as well in different patch.